pkg/network/common/common.go

```diff
@@ -4,27 +4,30 @@ import (
 	"fmt"
 	"net"
 	"reflect"
-	"strings"
-	"time"
 
 	"github.com/golang/glog"
 
-	configapi "github.com/openshift/origin/pkg/cmd/server/api"
-	networkapi "github.com/openshift/origin/pkg/network/apis/network"
-	networkclient "github.com/openshift/origin/pkg/network/generated/internalclientset"
-	"github.com/openshift/origin/pkg/util/netutils"
-
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
 	kerrors "k8s.io/apimachinery/pkg/util/errors"
-	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/apimachinery/pkg/watch"
 	kcache "k8s.io/client-go/tools/cache"
 	kapi "k8s.io/kubernetes/pkg/apis/core"
 	"k8s.io/kubernetes/pkg/apis/networking"
 	kinternalinformers "k8s.io/kubernetes/pkg/client/informers/informers_generated/internalversion"
+
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
+	networkapi "github.com/openshift/origin/pkg/network/apis/network"
+	networkinformers "github.com/openshift/origin/pkg/network/generated/informers/internalversion"
+	networkclient "github.com/openshift/origin/pkg/network/generated/internalclientset"
+	"github.com/openshift/origin/pkg/util/netutils"
 )
 
+type SDNInformers struct {
```

Comment:
I don't see a reason for this to exist, I'd prefer you remove it and simply pass the appropriate informer factory.

Reply:
@pravisankar look at https://github.com/danwinship/origin/commit/e4de1991e1db1c7e52d7c24363ee0aa53ad122ff. It's a rebase of a branch I'd been working on before, onto an earlier version of this PR. It gets rid of SDNInformers and RegisterSharedInformer and instead just calls the underlying funcs directly, like the rest of the code in origin does.

Reply:
@danwinship cool, I will pick the changes from your repo, thanks!